### DISCUSSION TITLE
Backup - tubeszb-cc2652-eth_usb

### DISCUSSION THREAD

**Discussion by zyberwoof** [tube0013/tube_gateways#142] [Q&A] [answered]
### Are there any tasks that should be done to backup the tubeszb-cc2652-eth_usb?

My setup includes Home Assistant, Mosquitto, and Zigbee2MQTT.  Each one is running in Docker containers.  And their configuration and data are stored in persistent volumes.  I frequently backup those volumes.

I wanted to check and see if there is anything special I should be doing to backup the tubeszb-cc2652-eth_usb itself.  Or does the z2m container already contain everything needed to restore or migrate the Zigbee network?

**Comment by tube0013** [comment, marked as answer]
z2m should create a back up each time the network is started - you should see a network-backup.json in the z2m config folder.

note if you migrate you will always want the most recent backup and it may be a good idea to up the tx_counter within to keep the network happy.
